### DISCUSSION TITLE
Unable to re-register and login after manually deleting the account

### DISCUSSION THREAD

**Discussion by zhanglei364** [vercel/community#3892] [Account Issue]
### Question

I would like to delete my account and re-login with GitHub to create a new project to test which step in the waline system setup went wrong, but after I deleted my account the website prompted me " This user account is blocked." I am currently unable to use GitHub to log into the website. I am currently unable to log into the website using GitHub, can you restore my account so that I can log into the vercel website using my GitHub account.

Translated with www.DeepL.com/Translator (free version)

**Comment by amyegan** [comment]
Closing as a duplicate of answered discussion https://github.com/vercel/community/discussions/3960
